images/runtime/python/template.Dockerfile

```diff
@@ -84,7 +85,7 @@ RUN pip install --upgrade pip \
     && pip install debugpy \
     && pip install viztracer==0.15.6 \
     && pip install vizplugins==0.1.3 \
-    && pip install orjson==3.8.1 \
+    && if [ "%PYTHON_VERSION%" != "3.12" ]; then pip install orjson==3.8.10; fi \
```

Comment:
Why do we need to install this library?

Reply:
I believe this one was added for user's convenience since this is a commonly used package (and also one of the fastest library to handle json). From quick global search, this package is not used within our repo anywhere else.